daft/dataframe/dataframe.py

```diff
@@ -859,6 +860,7 @@ def write_csv(
             write_mode (str, optional): Operation mode of the write. `append` will add new data, `overwrite` will replace the contents of the root directory with new data. `overwrite-partitions` will replace only the contents in the partitions that are being written to. Defaults to "append".
             partition_cols (Optional[List[ColumnInputType]], optional): How to subpartition each partition further. Defaults to None.
             io_config (Optional[IOConfig], optional): configurations to use when interacting with remote storage.
+            delimiter (Optional[str], optional): single character string to use as the delimiter. Defaults to None, which uses the default delimiter for CSV (",").
```

Comment:
```suggestion
            delimiter (str, optional): single character string to use as the delimiter. Defaults to use a comma (","): the default for CSV. Raises an error if the string is not exactly one character long.
            
Raises:
    ValueError: If the delimiter is not exactly one character long.
```

Suggested doc changes.